ugs-core/test/com/willwinder/universalgcodesender/GrblCommunicatorTest.java

```diff
@@ -356,7 +356,7 @@ public void testCancelSend() {
         instance.streamCommands();
         instance.cancelSend();
         // Verify that there are several active commands.
-        expectedBool = true;
+        expectedBool = false;
```

Comment:
Please update the test comment here, something like "Make sure cancelSend clears out the commands."